tools/switch.sh

```diff
@@ -27,7 +27,8 @@ Usage: $0 [-h] [-t] COMMAND [ARGS...]
 
 Commands:
     slot-info                            display slot status for each physical port
-    status                               display port status
+    status                               display port status for Rabbit connections
+    switchtec-status                     display switchtec utility's port status inf
```

Comment:
info?

Reply:
Since I'm likely the only one who will ever use this, I think I prefer slot-info to indication what will be coming out.